api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/OnArtForAtleastXmonthsCalculation.java

```diff
@@ -0,0 +1,75 @@
+package org.openmrs.module.eptsreports.reporting.calculation.dsd;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+import org.openmrs.api.context.Context;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.result.CalculationResult;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.calculation.result.SimpleResult;
+import org.openmrs.module.eptsreports.reporting.calculation.AbstractPatientCalculation;
+import org.openmrs.module.eptsreports.reporting.calculation.BooleanResult;
+import org.openmrs.module.eptsreports.reporting.calculation.generic.InitialArtStartDateCalculation;
+import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
+import org.openmrs.module.reporting.common.Age;
+import org.openmrs.module.reporting.common.Birthdate;
+import org.openmrs.module.reporting.data.person.definition.BirthdateDataDefinition;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OnArtForAtleastXmonthsCalculation extends AbstractPatientCalculation {
```

Comment:
So we have this same class under the PVLS folder - they seem to be doing the same thing. Can we look at using the same calculation?

Reply:
cc @ningosi to see if these two calculations are similar enough to use the same one

Reply:
We discussed this on the call. Probably something that we needed to do for the future as tech debt.